Assets/Shopify/UIToolkit/Test/Unit/Editor/TestThemeControllerBase.cs

```diff
@@ -193,10 +193,10 @@ public class MockThemeController : ThemeControllerBase {
                 yield return null;
             }
 
-            _themeController.UpdateVariantInCart(_variants.First(), 1);
-            _themeController.UpdateVariantInCart(_variants[1], 2);
+            _themeController.Cart.UpdateVariant(_variants.First(), 1);
+            _themeController.Cart.UpdateVariant(_variants[1], 2);
 
-            _themeController.ClearCart();
+            _themeController.Cart.ClearCart();
```

Comment:
One of the advantages of breaking apart the cart is that we can test its functionality as a separate unit... how hard would it be to pull these out into a TestCartController or something similar?

Reply:
Shouldn't be too tough and that's a good point.